cmd/spiffe/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	spiffe "github.com/spiffe/go-spiffe"
+)
+
+func main() {
+	// Make log not print out time information
+	log.SetFlags(0)
+
+	if len(os.Args) <= 1 {
+		log.Fatalf("\033[31mexpecting path [paths...] e.g\033[00m\n\tspiffe /var/lib/letsencrypt/spiffe.pem $HOME/Desktop/certs/cert.pem\033[00m\n")
+	}
+
+	seenPaths := make(map[string]bool)
```

Comment:
It seems that we can simplify this a bit by taking only one arg instead of many. I'm curious if you have a use for examining multiple certs in one invocation?